### PR TITLE
[DOCU-1590] Update Docker Installation docs

### PR DESCRIPTION
**Pull request by falondarville**
### Review

@lena-larionova 

### Summary

* update tag image section
* text changes

### Reason

Addresses Issue #2911 

### Testing

TBA

### CODE REVIEW

File: app/enterprise/2.4.x/deployment/installation/docker.md [hunk 1]
```diff
@@ -38,79 +38,67 @@ To complete this installation you will need a Docker-enabled system with proper
...
+
+{:.note}
+> The DNS or IP address for `KONG_PORTAL_GUI_HOST` should be preceded by a protocol, for example, `http://`.
 
 ## Step 6. Verify your installation
 
 1. Access the `/services` endpoint using the Admin API:
 
-    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{HOSTNAME}</div>:8001/services</code></pre>
+    <pre><code>curl -i -X GET --url http://<div contenteditable="true">{DNS_OR_IP}</div>:8001/services</code></pre>
```
Comment (lena-larionova) [inline app/enterprise/2.4.x/deployment/installation/docker.md:59]: Can you change all the `{DNS_OR_IP}` entries to `{HOSTNAME}` (or just `{HOST}`)?

**Review by lena-larionova — COMMENTED** [review]
Just one comment.

And you can ignore the broken Build and Test check, that's unrelated to your PR. It's something coming from Mesh (Jennifer's on it).

File: app/enterprise/2.4.x/deployment/installation/docker.md (hunk 1 above)
Comment (falondarville) [reply app/enterprise/2.4.x/deployment/installation/docker.md]: Changed.

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**